diff --git a/tf2/CMakeLists.txt b/tf2/CMakeLists.txt
index a43e14b..7bdbbe9 100644
--- a/tf2/CMakeLists.txt
+++ b/tf2/CMakeLists.txt
@@ -50,8 +50,8 @@ target_link_libraries(speed_test tf2  ${console_bridge_LIBRARIES})
 add_dependencies(tests speed_test)
 add_dependencies(tests ${catkin_EXPORTED_TARGETS})
 
-catkin_add_gtest(test_transform_datatypes test/test_transform_datatypes.cpp)
-target_link_libraries(test_transform_datatypes tf2  ${console_bridge_LIBRARIES})
-add_dependencies(test_transform_datatypes ${catkin_EXPORTED_TARGETS})
+catkin_add_gtest(test_transform2_datatypes test/test_transform_datatypes.cpp)
+target_link_libraries(test_transform2_datatypes tf2  ${console_bridge_LIBRARIES})
+add_dependencies(test_transform2_datatypes ${catkin_EXPORTED_TARGETS})
 
 endif()
